demos/math_bot.js

```diff
@@ -11,12 +11,11 @@ const mathjs = require('mathjs')
 //          /math sqrt(pi/2) * 3!`
 //
 
-const msgReply = function(s) {
+const msgReply = s => {
   let a1, a2, ans, b1, b2, eqn
   try {
     ans = '= ' + mathjs['eval'](s).toString()
-  } catch (error) {
-    console.error(error)
+  } catch {
```

Comment:
This is invalid JavaScript syntax, throwing an error.

Needs to be

```js
try { ... } catch (error) { ... }
```

I'm running `lts/carbon → 8.12.0.` and it's still invalid.